dspace-uclouvain/src/main/java/org/dspace/uclouvain/authority/configuration/UCLouvainAuthorAuthorityAPIConfiguration.java

```diff
@@ -1,15 +1,24 @@
 package org.dspace.uclouvain.authority.configuration;
 
 public class UCLouvainAuthorAuthorityAPIConfiguration {
-    private String filter;
+    private String authorFilter;
+    private String promoterFilter;
```

Comment:
Maybe it's better to implements a `Map<String, String> filters` solution to allow more filters without changing the code in next commit ? 
We could have a generic `getFilter(String keyName)` method and some shortcut `getPromotorFilter()`, ...